topics/http-client_multiplatform.md

```diff
@@ -78,24 +78,4 @@ dependencies {
 
 There is a full sample using the common client in the ktor-samples repository [mpp/client-mpp](https://github.com/ktorio/ktor-samples/tree/1.3.0/mpp/client-mpp).
```

Comment:
Shouldn't we change `1.3.0` to `master` after merging https://github.com/ktorio/ktor-samples/pull/89 ?